### DISCUSSION TITLE
Split button in different components

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#1201] [Proposals]
### Goal

Align mistica-web implementation and Figma libraries

### Context

Para ser fiel a Mística Web deberíamos de tratarlos como componentes diferentes

### Impact

Change figma libraries published components

### Discussion Questions

deberíamos separar los botones y que sean componentes diferentes para evitar mezclar props?

**Comment by yceballost** [comment]
`+` Tendríamos más control en las props de cada componente 
 `-` La design exp se puede resentir un poco, pero es mejor para la devexp

**Comment by yceballost** [comment]
We are going to split button component for 9.0.0
